package.json

```diff
@@ -104,8 +104,8 @@
     "zeebe-bpmn-moddle": "^0.10.0"
   },
   "peerDependencies": {
-    "bpmn-js": "8.x",
-    "diagram-js": "7.x",
+    "bpmn-js": "9.x",
```

Comment:
Just for my understanding, we need to add `bpmn-js@9` as peer dependency because of the needed `isPlane` util, right? Or is there another reason why it won't work with `bpmn-js@8` anymore?

Reply:
Yes, this is for the `isPlane` utility. I considered having a utility in this repo as well but then we have to consider changes in 2 repos if we want to make changes

Reply:
I was thinking whether we could make this backward compatible in some way, [as we did with the `isImplicit` method](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/render/BpmnPropertiesPanelRenderer.js#L172), so we could avoid a using `bpmn-js@9` in any case, and mark both versions as peer dependencies

```json
"bpmn-js": "^8.x || ^9.x"
```

Only if it's feasible of course 🙂 Wdyt?

Reply:
I'll look into it, it should absolutely be feasible from what I can see :+1:

Reply:
Added with [fbad6b9](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/567/commits/fbad6b9287a03a733f55c103271e18b3d9861bef). Can't use a `getDi` util either, because that's new in `9` as well :man_shrugging:

Reply:
@marstamm Consider having a test case around to verify the bpmn-js@8 support:

Install it as bpmn-js-legacy: npm:bpmn-js@8.

Create properties panel in an integration test and see that it does not blow up.

Reply:
Cf. https://stackoverflow.com/questions/56134857/how-to-install-npm-package-under-alias-or-different-name.

Reply:
Also consider not adding it. People can always upgrade.

Reply:
Adding the integration tests was easy enough, so I decided there is no reason not to add them. Cf. [3696d6e](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/567/commits/3696d6e5d16a073f658f5fcd8ed80ed1ffe45aa5)